docs/modules/integrate/pages/kafka-connect-connectors.adoc

```diff
@@ -117,4 +118,18 @@ xref:pipelines:submitting-jobs.adoc#submitting-a-job-using-a-java-client-or-embe
 
 == Fault Tolerance
 
-When a Kafka Connect Source connector is deployed to a Hazelcast cluster, snapshots of the connector state are stored in the Jet engine. For example, partition offsets and any metadata required for a restart or recovery. If there is a connector failure, the recovery behavior will depend on the type of connector. Refer to the documentation of your Kafka Connect Source connector for more detailed information. 
+When a Kafka Connect Source connector is deployed to a Hazelcast cluster, snapshots of the connector state are stored by the Jet engine.
+Example for such states are partition offsets and any metadata required for a restart or recovery. If there is a connector failure, the recovery behavior will depend on the type of connector. Refer to the documentation of your Kafka Connect Source connector for more detailed information.
+
+== Parallelism and reconfigurations
+
+Kafka Connect Source uses `tasks.max` configuration property to determine parallelism.
+Even if pipeline's total parallelism is higher than `tasks.max` value, the `tasks.max` value takes precedence.
+It is prohibited to have pipeline's total parallelism (edge's local parallelism * number of nodes) lower than `tasks.max`.
+
+After creation of an instance of a SourceConnector, Hazelcast Jet asks it to provide `tasks.max` task configurations. If the
+SourceConnector returns less than `tasks.max` configs, the remaining processors will become no-op.
```

Comment:
If the Source Connector returns ????, the remaining processors are ignored.

???? I wasn't sure what "less than `task.max` configs" meant; is it:

a value lower than the current `tasks.max` configuration

Reply:
>the remaining processors are ignored.
That wouldn't be true IMHO; processors are not ignored, they are running and waiting for reconfiguration, but main processing method will become no-op (will never return anything unless a reconfiguration is requested).

> I wasn't sure what "less than task.max configs" meant; is it:

Will reword this to:
```
If the SourceConnector returns fewer configurations than `tasks.max` value, the remaining processors will become no-op.
```